docs/about.rst

```diff
@@ -0,0 +1,57 @@
+**About**
+
+DefectDojo Basics
+~~~~~~~~~~~~~~~~~
+
+Terms
+*****
+There are several terms that will be helpful to understand as you work with DefectDojo
+
+Products
+********
+This is the name of any project, program, team, or company that you are currently testing.
+
+*Examples*
+	* OpenStack Neutron
+	* Internal wiki
+	* Hipchat
+
+Product types
+*************
+These can be business unit divisions, different offices or locations, or any other logical way of distinguishing "types" of products.
+
+*Examples*
+	* Internal / 3rd party
+	* Main company / Acquisition
+	* San Francisco / New York offices
+
+Engagements
+***********
+Engagements are moments in time when testing is taking place. They are associated with a name for easy refernece, a time line, a lead (the user account of the main person conducting the testing), a test strategy, and a status.
+
+*Examples*
+	* Beta
+	* Quarterly PCI Scan
+	* Release Version X
+
+Test Types
+**********
+Thes can be any sort of distinguishing characteristic about the type of testing that was done in an Engagement.
```

Comment:
"These" is misspelled.
I recommend changing "in" to "during" as in:
(These can be any sort of distinguishing characteristic about the type of testing that was done during an Engagement.)